plugins/vis/hu.bme.mit.gamma.plantuml.transformation/src/hu/bme/mit/gamma/plantuml/transformation/StatechartToPlantUmlTransformer.xtend

```diff
@@ -262,6 +262,11 @@ class StatechartToPlantUmlTransformer {
 		val state = statenode as State
 		if (!(state.entryActions.empty) || !(state.exitActions.empty)) {
```

Comment:
Isn't `!(state.invariants.empty)` missing from this condition?

Reply:
Thank you, I added the missing condition.